### PR TITLE
Add dns_bind_ip_blacklist and dns_forward_zones

### PR DESCRIPTION
**Pull request by nlsun**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by nlsun** [comment]
I've separated it into 2 commits, the second commit contains my changes.

**Comment by nlsun** [comment]
@joel-hamill ready for review

File: 1.10/installing/custom/configuration-parameters.md [hunk 1]
```diff
@@ -63,6 +63,27 @@ specified.
   This parameter specifies where to permanently store DC/OS packages. The value must be a file URL,
   for example, `file:///var/lib/dcos/cosmos/packages`.
 
+### dns_bind_ip_blacklist
+This parameter is a list of IP addresses that DC/OS DNS resolvers will not bind to.
+
+### dns_forward_zones
+This parameter is used by DC/OS DNS resolvers to configure the forwarding of queries for DNS addresses in a particular DNS zone to a custom DNS resolver.
```
Comment (joel-hamill) [inline 1.10/installing/custom/configuration-parameters.md]: How about: `This parameter specifies a YAML nested list (`-`) of DNS addresses and zones whose query requests are forwarded to a custom resolver.`

File: 1.10/installing/custom/configuration-parameters.md [hunk 2]
```diff
@@ -63,6 +63,27 @@ specified.
...
+
+This takes the form of nested lists. For example:
```
Comment (joel-hamill) [inline 1.10/installing/custom/configuration-parameters.md]: Think you can remove `This takes the form of nested lists.`

**Review by joel-hamill — CHANGES_REQUESTED** [review]

**Comment by nlsun** [comment]
@joel-hamill made the changes

**Review by joel-hamill — APPROVED** [review]

**Pull request merged**